ansible/group_vars/all

```diff
@@ -48,3 +48,40 @@ db:
   authkeys:
   - guest
   - whisk.system
+
+# The default value for catalog_source is empty, meaning that
+# openwhisk will not install the catalog from the catalog repositories.
+# If catalog_source is not set to "catalog-repos", the other variables
+# like namespace, api_host, auth_key_file and catalog_repos will not
+# be effective.
+catalog_source: ""
+
+# The default name space is /whisk.system. The namespace must begin with a slash "/".
+namespace: "/whisk.system"
+
+# The api_host is used to specify the IP address of the host, where
+# openwhisk edge host is running.
+api_host: "172.17.0.1"
```

Comment:
@rabbah I have found out the way to pick the value of the edge host in the yml file, so I can remove this variable.